SQL/Archive/17.1/2017-05-11-unique-sub-tests.sql

```diff
@@ -0,0 +1,8 @@
+WARNINGS;
+SET SQL_NOTES=1;
```

Comment:
@xlecours should the original value be saved to a variable first and reset after the patch ?